doc/latex.rst

```diff
@@ -189,14 +189,25 @@ The available styling options
     default ``true``. Tells whether long lines in :rst:dir:`code-block`\ 's
     contents should wrap.
 
+``literalblockcappos``
```

Comment:
This might be related with #1723. But the proper way is not discussed yet. So let's go ahead.